packages/cocoindex/tests/test_stop_mcp_proxy.py

```
@@ -0,0 +1,88 @@
+
+
+class TestMain:
+    def _invoke(self, payload: dict, monkeypatch: pytest.MonkeyPatch) -> str:
+        buffer = io.StringIO()
+        monkeypatch.setattr(sys, "stdin", io.StringIO(json.dumps(payload)))
+        monkeypatch.setattr(sys, "stdout", buffer)
+        stop_hook.main()
+        return buffer.getvalue()
+
+    def test_does_not_import_stop_proxy(self) -> None:
+        """stop-mcp-proxy.py は proxy_manager.stop_proxy を一切 import しない（EV-09）。"""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)